README.md

```diff
@@ -11,11 +11,14 @@ $ ./odoo-bin --upgrade-path=/path/to/upgrade-util/src,/path/to/other/upgrade/scr
 ```
 
 ### The alternative one
-On platforms where you dont manage odoo yourself, i.e. [Odoo.sh](https://www.odoo.sh/), you can install this package via pip:
+On platforms where you dont manage odoo yourself, you can install this package via pip:
 ```shell-session
 $ python3 -m pip install git+https://github.com/odoo/upgrade-util@master
 ```
-
+On [Odoo.sh](https://www.odoo.sh/) it is recommended to add it to the `requirements.txt` of your repository:
```

Comment:
I think we should create a section for Odoo.sh so we can link directly on it from the Odoo.sh UI and we know we can put specifics later if needed.

```md
### On Odoo.sh
```

@KangOl you're ok with this ?

Reply:
@sts-odoo what's the advantage of adding `odoo_upgrade` in front? I've added it without it, works fine
![image](https://github.com/odoo/upgrade-util/assets/84017456/eef1f3d1-ce69-420d-8548-530ad35be99e)